aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -1255,6 +1255,64 @@ void linalg_eigh_cusolver(Tensor& eigenvalues, Tensor& eigenvectors, Tensor& inf
   }
 }
 
+// The 'apply_' word is used for templated by dtype functions that call an API routine
+// underneath. Since the cusolver API has a slightly different structure we do not prepend
+// apply_ to this function.
+void lu_looped_cusolver(const Tensor& self, const Tensor& pivots, const Tensor& infos, bool get_pivots) {
+  // Fill the pivots tensor with indices using 1-based (Fortran) indexing. This
+  // is needed for maintaining the same results with MAGMA.
+  auto k = std::min(self.size(-2), self.size(-1));
+  Tensor pivots_tmp = at::arange(1, k + 1, self.options().dtype(at::kInt)).expand_as(pivots);
+  pivots.copy_(pivots_tmp);
+
+  AT_DISPATCH_FLOATING_TYPES(
+    self.scalar_type(),
+    "lu_cusolver",
+    [&self,
+     &pivots,
+     &infos,
+     &get_pivots]() {
```

Comment:
nit. Given that we are doing explicit capture, better to pass by value a `bool`.